test/spec/ol/format/mvt.test.js

```diff
@@ -82,4 +85,21 @@ where('ArrayBuffer').describe('ol.format.MVT', function() {
 
   });
 
+  describe('ol.featureloader.tile', function() {
```

Comment:
This is actually a test for the featureloader working with tile-pixels as units. It could be done with a mock format as well. So it makes sense to keep it in featureloader.test.js. But instead of an ol.format.MVT, you could use anything else that uses tile-pixels as units.